erdpy/contracts.py

```diff
@@ -1,14 +1,14 @@
 import base64
 import logging
-from typing import Any, List, Optional, Tuple
+from typing import Any, List, Optional, Tuple, Protocol
 
 from Cryptodome.Hash import keccak
 
 from erdpy import config, constants, errors
 from erdpy.accounts import Account, Address
-from erdpy.interfaces import IElrondProxy
 from erdpy.transactions import Transaction
 from erdpy.utils import Object
+from erdpy_network_providers.proxy_network_provider import ContractQuery
```

Comment:
Ideally, we should not import this, but define a similar class (that implements the `IContractQuery` interface) here in erdpy. Can stay as it is for the moment, though.

Reply:
Imported the `IContractQuery` interface and implemented a new `ContractQuery` class similar to the one in `erdpy-network-providers`.